.cursor/skills/waiting-for-build/scripts/query.py

```
@@ -149,6 +151,51 @@ def get_waiting_for_build_issues(sprint_id: int, headers: dict, base: str) -> li
+        if not r.ok:
+            return []
+        prs = []
+        for detail in r.json().get("detail", []):
+            for pr in detail.get("pullRequests", []):
+                if pr.get("status") == "MERGED":
+                    pr_id = pr.get("id", "")          # e.g. "#2427"
+                    m = re.search(r'(\d+)', pr_id)
+                    if m:
+                        prs.append(int(m.group(1)))
+        return prs
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)